Dockerfile-slim

```diff
@@ -0,0 +1,24 @@
+FROM node:0.10.44-slim
+
+# Use this to build a slimmer (~500MB) image for the app
+# use Dockerflow to build all the compiled assets and 
+# this one to bundle into an image for running
+
+WORKDIR /app
+
+# add a non-privileged user for installing and running
+# the application
+RUN groupadd --gid 10001 app && \
+    useradd --uid 10001 --gid 10001 --home /app --create-home app
+
+# install libgmp-dev for fast crypto and 
+RUN apt update && \
+    apt install -y libgmp-dev
+
+# docker 
+COPY . /app
+
+ENTRYPOINT ["npm"]
+CMD ["start"]
```

Comment:
Is `npm start` how moz-idp is started in production? Probably better to use the actual command, since `npm start` is typically how a dev instance should start.